self-tests.yml

```diff
@@ -809,7 +814,7 @@ buildvariants:
       - ubuntu1604-build
       - ubuntu1604-test
     expansions:
-      build_env: "GOROOT=/opt/go DISABLE_COVERAGE=yes EVERGREEN_ALL=true PATH=/opt/node/bin:$PATH"
+      build_env: "GOROOT=/opt/go DISABLE_COVERAGE=yes EVERGREEN_ALL=true"
```

Comment:
did you mean to revert this change? 

I think you want to specify a karama reporter here, right?

Reply:
This is intentional. I had this on the wrong build variant before